app/lib/analyzer/handlers.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:shelf/shelf.dart' as shelf;
+
+import '../shared/handlers.dart';
+
+/// Handlers for the analyzer service.
+Future<shelf.Response> analyzerServiceHandler(shelf.Request request) async {
+  logPubHeaders(request);
```

Comment:
As in the PR about search service, this is never invoked by the `pub` client itself and the other users of the API don't supply the `X-Pub-*` headers, so I think this can be safely removed.